src/components/Post.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import User from './User';
+import CommentList from './CommentList';
+
+class Post extends React.Component {
```

Comment:
Use a function instead of class.
you didn't use the potentiality of class
https://ru.reactjs.org/docs/components-and-props.html#function-and-class-components